src/param.rs

```diff
@@ -316,20 +320,23 @@ impl AudioNode for AudioParam {
 impl AudioParam {
     /// Current value of the automation rate of the AudioParam
     pub fn automation_rate(&self) -> AutomationRate {
-        self.raw_parts.shared_parts.load_automation_rate()
+        self.raw_parts.automation_rate
     }
 
     /// Update the current value of the automation rate of the AudioParam
     ///
     /// # Panics
     ///
     /// Some nodes have automation rate constraints and may panic when updating the value.
-    pub fn set_automation_rate(&self, value: AutomationRate) {
+    pub fn set_automation_rate(&mut self, value: AutomationRate) -> AutomationRate {
```

Comment:
Actually this is the behaviour in JS, but I'm not sure this is what we want here.

I guess, either we generalize this behavior to all "raw" setters, either we decide they don't return a value?

(same for `set_value` actually)